<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogModifPraticien</class>
 <widget class="QDialog" name="DialogModifPraticien">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Modification des praticiens</string>
  </property>
  <property name="windowIcon">
   <iconset resource="mesImages.qrc">
    <normaloff>:/images/leLogo</normaloff>:/images/leLogo</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(123,202,255);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Code :</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEditMatricule">
         <property name="styleSheet">
          <string notr="true">background-color:white;
border-width:1px;
border-color: rgb(21, 165, 255);
border-style:inset;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Nom :</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEditNom">
         <property name="styleSheet">
          <string notr="true">background-color:white;
border-width:1px;
border-color: rgb(21, 165, 255);
border-style:inset;</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Prénom :</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEditPrenom">
         <property name="styleSheet">
          <string notr="true">background-color:white;
border-width:1px;
border-color: rgb(21, 165, 255);
border-style:inset;</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Adresse :</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="lineEditAdresse">
         <property name="styleSheet">
          <string notr="true">background-color:white;
border-width:1px;
border-color: rgb(21, 165, 255);
border-style:inset;</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Code Postal :</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="lineEditCP">
         <property name="styleSheet">
          <string notr="true">background-color:white;
border-width:1px;
border-color: rgb(21, 165, 255);
border-style:inset;</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Ville :</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="lineEditVille">
         <property name="styleSheet">
          <string notr="true">background-color:white;
border-width:1px;
border-color: rgb(21, 165, 255);
border-style:inset;</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Coefficient de notoriété :</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Type de praticien :</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QComboBox" name="comboBoxTypeModif">
         <property name="styleSheet">
          <string notr="true">background-color:white;
border-width:1px;
border-color: rgb(21, 165, 255);
border-style:inset;</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="lineEditCN">
         <property name="styleSheet">
          <string notr="true">background-color:white;
border-width:1px;
border-color: rgb(21, 165, 255);
border-style:inset;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonOk">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:white;
border-width:1px;
border-color: rgb(21, 165, 255);
border-style:outset;</string>
       </property>
       <property name="text">
        <string>&amp;Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonAnnuler">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:white;
border-width:1px;
border-color: rgb(21, 165, 255);
border-style:outset;</string>
       </property>
       <property name="text">
        <string>&amp;Annuler</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="mesImages.qrc"/>
 </resources>
 <connections/>
</ui>
